modules/services/repositories/src/main/java/au/com/shiftyjelly/pocketcasts/repositories/di/RepositoryModule.kt

```diff
@@ -70,6 +72,10 @@ abstract class RepositoryModule {
     @Singleton
     abstract fun providesSubscriptionManager(subscriptionManagerImpl: SubscriptionManagerImpl): SubscriptionManager
 
+    @Binds
+    @Singleton
+    abstract fun providesSyncAccountManager(syncAccountManagerImpl: SyncAccountManagerImpl): SyncAccountManager
```

Comment:
For the tests, I needed SyncAccountManager to have an interface or if I mocked the object it still called the method and crashed the tests.